build.gradle

```diff
@@ -90,4 +95,10 @@ tasks.coveralls {
     doFirst {
         coveralls.sourceDirs = subprojects.sourceSets.main.java.srcDirs.flatten()
     }
+}
+
+buildScan {
```

Comment:
This provides detailed stats on builds whenever we need. When I have a passing PR build ill post a report here as an example

Reply:
https://gradle.com/s/vjdwtfo64blnm